README.md

```diff
@@ -249,6 +250,27 @@ kubeaudit np
 WARN[0000] Default allow mode on test/testing
 ```
 
+<a name="resources" />
+
+### Audit resources
+
+It checks that every resource has a CPU and memory limit. See [Kubernetes Resource Quotas](https://kubernetes.io/docs/concepts/policy/resource-quotas/)
```

Comment:
Maybe its better to link [this](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/) instead of the resourcequota docs